core/src/main/java/org/frankframework/senders/RuntimeSenderWrapper.java

```diff
@@ -0,0 +1,93 @@
+/*
+   Copyright 2024 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package org.frankframework.senders;
+
+import org.frankframework.configuration.ConfigurationException;
+import org.frankframework.core.ISender;
+import org.frankframework.core.PipeLineSession;
+import org.frankframework.core.SenderException;
+import org.frankframework.core.SenderResult;
+import org.frankframework.core.TimeoutException;
+import org.frankframework.statistics.HasStatistics;
+import org.frankframework.statistics.StatisticsKeeperIterationHandler;
+import org.frankframework.stream.Message;
+
+import lombok.Setter;
+
+/**
+ * Wrapper for senders, that opens 'runtime' before each sender action, and closes afterwards.
```

Comment:
I still think that this change makes the documentation clearer:

```suggestion
 * Wrapper for senders, that opens the wrapped sender at runtime before each sender action, and closes it afterwards.
```